<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>493</width>
    <height>496</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="currentTimeFrameLayout">
      <item>
       <widget class="QLabel" name="currentTimeLabel">
        <property name="text">
         <string>Current Time</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CustomTimeEdit" name="currentTimeEdit">
        <property name="displayFormat">
         <string>HH:mm:ss.zzz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="currentFrameLabel">
        <property name="text">
         <string>Current Frame</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="currentFrameBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000000</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="annotLabel">
      <property name="text">
       <string>annotation label</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="AnnotationsView" name="annotationsView">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="resizeAnchor">
       <enum>QGraphicsView::AnchorViewCenter</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="controlButtonLayout">
      <item>
       <widget class="QPushButton" name="toStartButton">
        <property name="text">
         <string>|&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fbButton">
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="playButton">
        <property name="text">
         <string>Play/Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stepButton">
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ffButton">
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="toEndButton">
        <property name="text">
         <string>&gt;|</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="nextPrevLayout">
      <item>
       <widget class="QPushButton" name="previousButton">
        <property name="text">
         <string>Previous Annotation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextButton">
        <property name="text">
         <string>Next Annotation</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="playbackSpeedLayout">
      <item>
       <widget class="QPushButton" name="halveFrameRateButton">
        <property name="text">
         <string>0.5x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="oneXFrameRateButton">
        <property name="text">
         <string>1x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="doubleFrameRateButton">
        <property name="text">
         <string>2x</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="mainButtonLayout">
      <item>
       <widget class="QComboBox" name="channelComboBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="newChannelPushButton">
        <property name="text">
         <string>New Channel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="channelHorizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="trialPushButton">
        <property name="text">
         <string>Select Trial...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quitButton">
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnnotationsView</class>
   <extends>QGraphicsView</extends>
   <header>widgets.annotationsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>CustomTimeEdit</class>
   <extends>QTimeEdit</extends>
   <header location="global">timeEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
